src/stack.ts

```diff
@@ -131,6 +131,16 @@ export function makeHandler(stack: Stack) {
         stack.pop();
         await waitSlide();
       }
+      if (stack.stack.value.length === 0) {
+        // Update URL without reloading
+        history.pushState(history.state, '', toPath);
+
+        // Use a timeout to simulate a navigation update
+        setTimeout(() => {
+          // Force Vue Router to handle the route change
+          window.dispatchEvent(new Event('popstate'));
+        }, 0);
+      }
```

Comment:
@hbhat-zenken  

`handleUrlChange` is called when the URL is changed.

Therefore, dispatching a 'popstate' event in `handlerUrlChange' seems to cause handlerUrlChange to be called again, resulting in an infinite loop.

Try to add console.log in the line before 'if (stack.stack.value.length === 0)').

Reply:
@hbhat-zenken Sorry, I mistake quick check this code.

works perfectly!!

Reply:
@mu-iridge Thanks!!!